opencensus/stats/stats.py

```diff
@@ -12,24 +12,29 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from datetime import datetime
+
+from opencensus.metrics.export.metric_producer import MetricProducer
 from opencensus.stats.stats_recorder import StatsRecorder
 from opencensus.stats.view_manager import ViewManager
 
 
-class Stats(object):
+class Stats(MetricProducer):
     """Stats defines a View Manager and a Stats Recorder in order for the
     collection of Stats
     """
+
     def __init__(self):
-        self._stats_recorder = StatsRecorder()
-        self._view_manager = ViewManager()
-
-    @property
-    def stats_recorder(self):
```

Comment:
I'd understand making an attribute protected and then exposing it as a property if we were returning a copy, but we don't seem to be doing that here.

Reply:
https://github.com/census-instrumentation/opencensus-python/issues/477